examples/seldon_core_setup.ipynb

```diff
@@ -0,0 +1,225 @@
+   "source": [
+    "!kubectl create clusterrolebinding kube-system-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install Ingress\n",
+    "\n",
+    "### Ambassador\n",
+    "\n",
+    "[Ambassador install](https://docs.seldon.io/projects/seldon-core/en/latest/workflow/install.html#install-ambassador)\n",
+    "\n",
+    "**Note:** There are reported gRPC issues with ambassador (see https://github.com/SeldonIO/seldon-core/issues/473)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Check all services are running before proceeding."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Waiting for deployment \"ambassador\" rollout to finish: 0 of 3 updated replicas are available...\n",
+      "Waiting for deployment \"ambassador\" rollout to finish: 1 of 3 updated replicas are available...\n",
+      "Waiting for deployment \"ambassador\" rollout to finish: 2 of 3 updated replicas are available...\n",
+      "deployment \"ambassador\" successfully rolled out\n"
+     ]
+    }
+   ],
+   "source": [
+    "!kubectl rollout status deployment.apps/ambassador"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Forwarding from 127.0.0.1:8003 -> 8080\r\n",
+      "Forwarding from [::1]:8003 -> 8080\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "!kubectl port-forward $(kubectl get pods -n seldon -l app.kubernetes.io/name=ambassador -o jsonpath='{.items[0].metadata.name}') -n seldon 8003:8080"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*or*\n",
+    "\n",
+    "### Istio\n",
+    "\n",
+    "[Istio install](https://docs.seldon.io/projects/seldon-core/en/latest/workflow/install.html#install-istio-ingress-gateway)\n",
+    "\n",
+    "**Note:** Remember to add ```--set istio.enabled=true``` flag when installing Seldon Core as shown below."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install Seldon Core\n",
+    "\n",
+    "Follow the [Seldon Core Install documentation](https://docs.seldon.io/projects/seldon-core/en/latest/workflow/install.html)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note:** If using Istio Ingress remember to add ```--set istio.enabled=true``` flag."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!helm install seldon-core seldon-core-operator --repo https://storage.googleapis.com/seldon-charts --set usageMetrics.enabled=true --namespace seldon-system --set istio.enabled=true"
```

Comment:
It seems that the Seldon Core installation command is setting `--set istio.enabled=true`. Should we remove it to default to Ambassador?

Reply:
Have removed helm install command from setup notebook to reduce confusion so that docs are "single source of truth" for `helm install` command

Reply:
Just to clarify, I think that if we have `helm install seldon-core ...` without the `--set istio.enabled=true` flag is probably alright as well. We use Ambassador for most examples, so it's probably good to default to it (instead of istio).

This is also a good solution though!

Reply:
Sure! I had only put the `helm install...` command in there to show how to use the `--set istio...` flag... but it seems that is confusing as it looks like I've put it in to show how to use the install command generally - so for that reason I think it's better to just take it out altogether and reference the install docs which shows how to use it in both instances.